mifospay/src/main/java/org/mifos/mobilewallet/mifospay/invoice/presenter/InvoicePresenter.kt

```diff
@@ -28,7 +28,7 @@ class InvoicePresenter @Inject constructor(
     }
 
     override fun getInvoiceDetails(data: Uri?) {
-        mUseCaseHandler.execute(fetchInvoiceUseCase, FetchInvoice.RequestValues(data),
+        mUseCaseHandler.execute(fetchInvoiceUseCase, data?.let { FetchInvoice.RequestValues(it) },
```

Comment:
Make above changes and remove let and pass directly `FetchInvoice.RequestValues(it)`